app/posts/publish-teacher-training-courses/2023-05-18-adding-study-sites.md

```diff
@@ -0,0 +1,200 @@
+date: 2023-05-18
+tags:
+  - locations
+  - study sites
+related:
+  items:
+    - text: Improving how providers add schools
+      href: /publish-teacher-training-courses/improving-how-providers-add-schools/
+screenshots:
+  items:
+    - text: Study sites list
+      src: study-sites--list.png
+    - text: Study sites list - empty
+      src: study-sites--list-empty.png
+    - text: Adding a study site
+      src: study-sites--find.png
+    - text: Adding a study site - autocomplete
+      src: study-sites--find-autocomplete.png
+    - text: Adding a study site - error
+      src: study-sites--find-error.png
+    - text: Editing study site details
+      src: study-sites--edit.png
+    - text: Check your answers
+      src: study-sites--check-your-answers.png
+    - text: Study site successfully added
+      src: study-sites--success.png
+    - text: Adding a study site manually
+      src: study-sites--form.png
+    - text: Adding a study site manually - error
+      src: study-sites--form-error.png
+    - text: Study site details
+      src: study-sites--details.png
+    - text: Remove study site
+      src: study-sites--remove.png
+    - text: Study site removed
+      src: study-sites--removed.png
+    - text: Study site cannot be removed
+      src: study-sites--cannot-remove.png
+---
+
+On a course page on Find postgraduate teacher training (Find), we show a list of schools where candidates could be placed to do their classroom training. During recent rounds of research, we found that there was a need for candidates to know where they will do the theoretical part of their training, too.
+
+We found that:
+
+-  it was important for candidates to know where they will do the theoretical part of their studies to make a more informed choice about what course to apply for
+- providers have more than one ‘Study site’ per course
+
+## What we changed
+
+We have added a section in Publish teacher training courses (Publish) called ‘Study sites’.
+
+This section allows providers to list all the places trainees may need to attend for theoretical training, such as a university campus.
+
+## How it works
+
+### Study site list
+
+On the study site list page, we show a:
+
+- button to ‘Add study site’
+- list of study sites the provider has added
+
+The list of study sites includes the:
+
+- Study site name
+- unique reference number (URN)
+
+If the user has not entered a URN for a study site, we show ‘Not entered’ in the URN column.
+
+If the provider does not have any study sites, we do not show an empty list. Instead, we show the message:
+
+>A study site, such as a university campus, is where trainees do theoretical training. Add study sites for your organisation and then link one or more to a course from the ‘Basic details’ tab on the course page.
+>
+>Candidates will see a list of the selected study sites in the ‘Training locations’ section on the course page on Find.
+>
+>Find out more about adding training locations to a course.
```

Comment:
Can we add a placeholder link here or put (link to be confirmed) after it or something? Just looks odd not having the link.

Reply:
I've added a dummy link to example.com